scripts/west_commands/runners/intel_adsp.py

```diff
@@ -63,14 +47,6 @@ def capabilities(cls):
     def do_add_parser(cls, parser):
         parser.add_argument('--remote-host',
                             help='hostname of the remote targeting ADSP board')
-        parser.add_argument('--rimage-tool', default=DEFAULT_RIMAGE_TOOL,
```

Comment:
removing all those options makes it less flexible and breaks existing users depending on those options.